cpp/src/replace/replace.cu

```diff
@@ -15,17 +16,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 #include <thrust/device_ptr.h>
 #include <thrust/find.h>
 #include <thrust/execution_policy.h>
 
 #include "cudf.h"
+#include "rmm/rmm.h"
+#include <copying.hpp>
+#include <types.hpp>
 #include "utilities/error_utils.hpp"
 #include "utilities//type_dispatcher.hpp"
+#include <utilities/cuda_utils.hpp>
 #include "utilities/cudf_utils.h"
+#include "bitmask/bit_mask.cuh"
 #include "bitmask/legacy_bitmask.hpp"
 
+using bit_mask::bit_mask_t;
```

Comment:
Avoid a `using namespace` at global scope like this. Limit it to block level scopes.